django/performances/models.py

```
@@ -2,6 +2,8 @@
 from django.db import models
 from projects.models import Project
 from django.conf import settings
+from django.core.files.storage import default_storage
+from django.template.defaultfilters import slugify
```

Ruff

`django.template.defaultfilters.slugify` imported but unused

Remove unused import: `django.template.defaultfilters.slugify`

(F401)